apps/explorer/src/components/OwnedObjects/ThumbNailsOnlyView.tsx

```diff
@@ -0,0 +1,79 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiObjectResponse } from '@mysten/sui.js/client';
+import { formatAddress } from '@mysten/sui.js/utils';
+import { Placeholder, Text } from '@mysten/ui';
+
+import { useResolveVideo } from '~/hooks/useResolveVideo';
+import { ObjectLink } from '~/ui/InternalLink';
+import { ObjectVideoImage } from '~/ui/ObjectVideoImage';
+import { parseObjectType } from '~/utils/objectUtils';
+import { trimStdLibPrefix } from '~/utils/stringUtils';
+
+function ThumbNail({ obj }: { obj: SuiObjectResponse }) {
+	const video = useResolveVideo(obj);
+	const displayMeta = obj.data?.display?.data;
+	const src = displayMeta?.image_url || '';
+	const name = displayMeta?.name ?? displayMeta?.description ?? '--';
+	const type = trimStdLibPrefix(parseObjectType(obj));
+	const id = obj.data?.objectId;
+
+	return (
+		<div>
+			<ObjectLink
+				display="flex"
+				objectId={id!}
+				label={
+					<div className="group relative">
+						<ObjectVideoImage
+							disabled
+							title={name}
+							subtitle={type}
+							src={src}
+							video={video}
+							variant="medium"
+							open={false}
+							setOpen={() => {}}
+						/>
+						<div className="absolute bottom-2 left-1/2 hidden w-10/12 -translate-x-1/2 justify-center rounded-lg bg-white/80 p-1 backdrop-blur-xl group-hover:flex">
+							<Text variant="subtitle/medium" color="steel-dark">
+								{formatAddress(id!)}
+							</Text>
+						</div>
+					</div>
+				}
+			/>
+		</div>
+	);
+}
+
+function ThumbNailsOnlyLoading({ limit }: { limit: number }) {
+	return (
+		<>
+			{new Array(limit).fill(0).map((_, index) => (
+				<div key={index} className="h-16 w-16 md:h-31.5 md:w-31.5">
+					<Placeholder rounded="lg" height="100%" />
+				</div>
+			))}
+		</>
+	);
+}
+
+interface ThumbNailsOnlyViewProps {
+	limit: number;
+	data?: SuiObjectResponse[];
+	loading?: boolean;
+}
+
+export function ThumbNailsOnlyView({ data, loading, limit }: ThumbNailsOnlyViewProps) {
+	return (
+		<div className="flex flex-row flex-wrap gap-2 overflow-auto md:gap-4">
+			{loading ? (
+				<ThumbNailsOnlyLoading limit={limit} />
+			) : (
+				data?.map((obj) => <ThumbNail key={obj.data?.objectId} obj={obj} />)
```

Comment:
nit: `Thumbnail` looks more correct to me

Reply:
That's true, pardon English